### PR TITLE
Use a module for the VPC flow logs

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request uses a Terraform module to create the VPC flow logs instead of using custom code.

## 💭 Motivation and Context

We can use this module everywhere instead of having custom code everywhere.

## 🧪 Testing

Successfully applied to the staging environment.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
LGTM after a quick review of the [module code](https://github.com/trussworks/terraform-aws-vpc-flow-logs) (looks pretty similar to what we had), but some testing here would be nice before we merge this thang.  🌵

**Review by hillaryj — APPROVED** [review]
👍

**Pull request merged**
